docs/docs/development/new-shield.md

```diff
@@ -11,14 +11,18 @@ import Metadata from "@site/src/data/hardware-metadata.json";
 
 ## Overview
 
-This guide will walk through the steps necessary to add ZMK support for a keyboard the uses a (Pro Micro compatible) addon MCU board to provide the microprocessor.
+This guide will walk through the steps necessary to add ZMK support for a keyboard the uses an addon MCU board (e.g. Pro Micro compatible) to provide the microprocessor.
```

Comment:
I am aware some of these are unchanged from the previous version but might as well fix them while we are touching them?

```suggestion
This guide will walk through the steps necessary to add ZMK support for a keyboard that uses an add-on MCU board (e.g. Pro Micro compatible) to provide the microprocessor.
```